examples/typescript/transfer_coin.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * This example shows how to use the Aptos client to create accounts, fund them, and transfer between them.
+ * Similar to ./simple_transfer.ts, but uses transferCoinTransaction to generate the transaction.
+ */
+
+import { Account, AccountAddress, Aptos, AptosConfig, Network } from "aptos";
+
+const COIN_STORE = `0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>`;
+const ALICE_INITIAL_BALANCE = 100_000_000;
+const BOB_INITIAL_BALANCE = 1_000_000;
+const TRANSFER_AMOUNT = 1_000_000;
+
+/**
+ * Prints the balance of an account
+ * @param aptos
+ * @param name
+ * @param address
+ * @returns {Promise<*>}
+ *
+ */
+const balance = async (aptos: Aptos, name: string, address: AccountAddress) => {
+  let resource = await aptos.getAccountResource({ accountAddress: address.toUint8Array(), resourceType: COIN_STORE });
+  let amount = Number((resource.data as { coin: { value: string } }).coin.value);
+
+  console.log(`${name}'s balance is: ${amount}`);
+  return amount;
+};
+
+const example = async () => {
+  console.log(
+    "This example will create two accounts (Alice and Bob), fund them, and transfer between them using transferCoinTransaction.",
+  );
+
+  // Setup the client
+  const config = new AptosConfig({ network: Network.DEVNET });
```

Comment:
devnet is default for now - might want to change it
```suggestion
  const config = new AptosConfig();
```